### ISSUE TITLE
tutorial/rf208,210,211,512 must be disabled without FFT

### ISSUE THREAD

**Issue by Axel-Naumann** [root-project/root#8261]
### Describe the bug

```   1018 - tutorial-roofit-rf208_convolution-py (Failed)
    1020 - tutorial-roofit-rf210_angularconv-py (Failed)
    1021 - tutorial-roofit-rf211_paramconv-py (Failed)
    1052 - tutorial-roofit-rf512_wsfactory_oper-py (Failed)
```
if ROOT is not built with fft3.

### Expected behavior

The tutorials are skipped.

### To Reproduce

Configure with testing, without fft, run tests.

### Additional context
Add to `fftw3_veto` in `tutorials/CMakeLists.txt`

**Comment by guitargeek** [comment]
Hi! I opened a PR for this yesterday:
https://github.com/root-project/root/pull/8248
I assigned you and Lorenzo as reviewers now.

**Comment by guitargeek** [comment]
Closed by #8248.

**Comment by guitargeek** [comment]
Added the `Fixed in: not applicable` project since this issue never affected any release (only temporarily affected `master` in the 6.26 development cycle).
